### ISSUE TITLE
Add *.orig to gitignore

### ISSUE THREAD

**Issue by shikharvaish28** [coala/community#194]
At present, ```.orig``` is not ignored therefore we need to add this to ```gitignore``` so that all files of these types are not included in commits.

Would be a good newcomer issue.

**Comment by li-boxuan** [comment]
This issue has been discussed in coala-bears repo
c.f. https://github.com/coala/coala-bears/issues/2510#issuecomment-396096183
The ideal solution is https://gitlab.com/coala/mobans/issues/1

**Comment by PandaBoi** [comment]
If this issue is open can I be assigned to it?

**Comment by AkshJain99** [comment]
@li-boxuan assign me this issue

**Comment by li-boxuan** [comment]
>  assign me this issue

No, see https://github.com/coala/community/issues/194#issuecomment-422658089

**Comment by AkshJain99** [comment]
Then I think we can close this issue
